packages/components/src/tabs/test/index.tsx

```diff
@@ -139,6 +139,13 @@ const ControlledTabs = ( {
 const getSelectedTab = async () =>
 	await screen.findByRole( 'tab', { selected: true } );
 
+const getSelectedTabPanelContent = async () => {
+	// While all TabPanels are rendered, only the active one will be found.
+	// The rest are all `hidden` and will be ignored by `findByRole()`.
+	const currentTabPanel = await screen.findByRole( 'tabpanel' );
+	return currentTabPanel.textContent;
+};
+
```

Comment:
I'm curious if we need a helper function and for it to be outside the new test, since it's only used there. Otherwise, I think it could be removed and simplified to one of these assertions: 

```
expect( screen.getByRole( 'tabpanel' ) ).toHaveTextContent(
			'Selected tab: Alpha'
);
```
Or
```
expect( screen.getByText( 'Selected tab: Alpha' ) ).toBeVisible();
```

Which would make the query and assertion a bit more specific. What do you think?

Reply:
Excellent, my brain was stuck on `findByRole` for some reason, which I needed to resovle before I could check `textContent`, which is what prompted me to abstract it into a helper. Your approach is much cleaner 🙂